lib/utils/__tests__/isStandardSyntaxFunction.test.js

```diff
@@ -1,42 +1,51 @@
 'use strict';
 
 const isStandardSyntaxFunction = require('../isStandardSyntaxFunction');
-const postcss = require('postcss');
 const valueParser = require('postcss-value-parser');
 
 describe('isStandardSyntaxFunction', () => {
 	it('calc', () => {
-		expect(isStandardSyntaxFunction(func('a { prop: calc(a + b) }'))).toBe(true);
+		expect(isStandardSyntaxFunction(getFunction('calc(a + b)'))).toBe(true);
 	});
 
 	it('url', () => {
-		expect(isStandardSyntaxFunction(func("a { prop: url('x.css') }"))).toBe(true);
+		expect(isStandardSyntaxFunction(getFunction("url('x.css')"))).toBe(true);
 	});
 
 	it('scss list', () => {
-		expect(isStandardSyntaxFunction(func('a { $list: (list) }'))).toBe(false);
+		// as in $list: (list)
+		expect(isStandardSyntaxFunction(getFunction('(list)'))).toBe(false);
 	});
 
 	it('scss map', () => {
-		expect(isStandardSyntaxFunction(func('a { $map: (key: value) }'))).toBe(false);
+		// as in $map: (key: value)
+		expect(isStandardSyntaxFunction(getFunction('(key: value)'))).toBe(false);
 	});
 
-	it('scss function in custom prop', () => {
-		expect(isStandardSyntaxFunction(func('a { --primary-color: #{darken(#fff, 0.2)} }'))).toBe(
-			false,
-		);
+	it('scss function in scss interpolation', () => {
+		expect(isStandardSyntaxFunction(getFunction('#{darken(#fff, 0.2)}'))).toBe(false);
+	});
+
+	it('CSS-in-JS interpolation', () => {
+		expect(
+			isStandardSyntaxFunction(
+				getFunction('${({ size }) => (size === "small") ? "0.8em" : "1em"}'),
```

Comment:
this is great, exactly what i was getting at in the other issue - a generic test for all css-in-js function nodes.

nice work 🎉